Docs/development/r-unit-test-guide.md

```diff
@@ -211,13 +210,13 @@ test_that("Binomial Test table results match", {
 test_that("0 plot matches", {
 	plotName <- results[["results"]][["containerPlots"]][["collection"]][["containerPlots_contBinom"]][["collection"]][["containerPlots_contBinom_0"]][["data"]]
 	testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
-	jaspTools::expect_equal_plots(testPlot, "0", dir = "BinomialTest")
+	jaspTools::expect_equal_plots(testPlot, "0")
```

Comment:
hmm... is "0" a good filename? It probably works fine but maybe in the example, we should be a bit more descriptive?

Reply:
No, it's a terrible filename, so on line 223 we have
```
You will be able to copy-paste the output directly into a test file. Just make sure that each test/expectation title makes some sense (e.g., "Binomial Test table results match" makes sense but "1 plot matches" could use some work).
In the example above we should adjust the two plot tests, because "0" and "1" are not very informative:
```

This is hidden by the diff viewer though.